backend/account.py

```diff
@@ -0,0 +1,50 @@
+from app import app, users_db
+from flask import request
+import flask_login as fl
+from bson.objectid import ObjectId
+import re  # regex
+
+
+# user class
+class User(fl.UserMixin):
+    def __init__(self, name) -> None:
+        super().__init__()
+        self.id = name
+
+
+@app.route("/change-email", methods=["POST"])
+@fl.login_required
+def change_email():
+    id = fl.current_user.id
+    new_email = request.form["new_email"]
+
+    filter = {"_id": ObjectId(str(id))}
+    update = {"$set": {"email": new_email}}
+    users_db.update_one(filter, update)
+    return "Success", 200
+
+
+@app.route("/change-password", methods=["POST"])
+@fl.login_required
+def change_password():
+    id = fl.current_user.id
+    new_password = request.form["new_password"]
+    if(len(new_password.strip()) == 0):
+        return "Password has no non-whitespace symbols", 
```

Comment:
Here I can see that you are not returning any http error code, it is done on purpose?